sdks/python/apache_beam/dataframe/doctests_test.py

```diff
@@ -212,13 +228,21 @@ def test_not_implemented_followed_by_name_error(self):
     self.assertEqual(result.attempted, 6)
     self.assertEqual(result.failed, 1)  # Only the very last one.
 
+  def test_failed_assignment(self):
+    result = doctests.teststring(
+        FAILED_ASSIGNMENT,
+        optionflags=doctest.ELLIPSIS,
+        not_implemented_ok=True)
+    self.assertNotEqual(result.attempted, 0)
+    self.assertEqual(result.failed, 0)
+
   def test_rst_ipython(self):
     try:
       import IPython
     except ImportError:
       raise unittest.SkipTest('IPython not available')
     result = doctests.test_rst_ipython(RST_IPYTHON, 'test_rst_ipython')
-    self.assertEqual(result.attempted, 5)
+    self.assertEqual(result.attempted, 8)
```

Comment:
Why did this change? There still seem to be only 5 ipython statements in the `RST_IPYTHON` rst.

Reply:
I added 

```
foo(
    4
)
```

plus the re-definition of foo and another invocation `foo(5)`.

Reply:
OK, thanks. I misunderstood how counting worked here.